Lib/glyphsLib/classes.py

```diff
@@ -3760,6 +3765,46 @@ def _bracket_info(self):
         else:
             return axis_index, bracket_crossover, None
 
+    def _is_brace_layer(self):
+        if self.parent.parent.format_version > 2:
+            return "coordinates" in self.attributes  # Glyphs 3
+        # Glyphs 2
+        return "{" in self.name and "}" in self.name and ".background" not in self.name
```

Comment:
or should it be `self.name.endswith(".background")`?

Reply:
I copied the existing code and tried not to change it since I don't know exactly what it is matching.